docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -125,6 +125,15 @@ Include it in your module file like this:
     # ... snip ...
     '''
 
+.. note::
```

Comment:
I think this needs moving earlier in the file and not as a `.. note::`
This document should be in the same order as the module file itself